learningequality/kolibri-design-system
# Added DocsExample

Description
I’ve added the DocsExample component that allows toggling between an example and its code.

Issue Addressed
Fixes #826.

Changelog
-Description: Added DocsExample component to toggle between example and code.
-Products Impact: -
-Addresses: #826.
-Components: DocsExample.

Guidance: The new component enhances the documentation UI by toggling code visibility.

Steps to Test
1.Navigate to the KCardGrid documentation page.
2.Verify that the example and code toggle correctly with a slide-up/slide-down animation.

Implementation Notes
-The DocsExample component uses a #code slot to accept code snippets in multiple languages.
-Slide toggle animation is applied when showing/hiding the code.
